main.cpp

```diff
@@ -35,319 +84,334 @@ send me a DM to check your pull request
  Wait for my code review.
  */
 
-/*
-1)knife
-5 properties:
-    1)pointiness
-    2)sharpness
-    3)thickness
-    4)length
-    5)price
-3 things it can do:
-    1)cut
-    2)stab
-    3)unscrew
- */
 
 struct Knife 
 {
-    // angle of knife point in degrees
+    
     float pointiness = 0.85f; 
-    // smallest thickness in mm
+ 
     float sharpness = 0.3f;
-    // biggest thickness
+
     float thickness = 1.5f;
-    //length in mm
+
     unsigned int length = 150;
-    //
+    
     float price = 75;
     
     struct Incision
     {
         float incisionDepth = 10;
         float incisionLength = 50;
 
-        void hurt(int pain);
+        void hurt(float pain);
     };
 
-    //cut
     void cut(Incision prettyNastyCut);
-    //stab
+  
     void stab();
-    //unscrew
+    
     void unscrew();
 };
 
-/*
-2)light
-5 properties:
-    1)temperature
-    2)luminosity
-    3)electrical consumption
-    4)size
-    5)price
-3 things it can do:
-    1)turn on
-    2)set the mood
-    3)elevate the production's quality
- */
+void Knife::Incision::hurt(float pain)
+{
+    pain = incisionDepth + incisionLength;
+}
+
+void Knife::cut(Incision prettyNastyCut)
+{
+    prettyNastyCut.hurt(1);
+}
+
+void Knife::stab()
+{
+    Incision newIncision;
+}
```

Comment:
This variable declaration triggers a warning.
since you declared the variable, use it in a `std::cout` statement like you did with `Knife::unscrew`.
using the variable will clear the warning.
- [ ] revise

Reply:
gave me a lengthy error, just used the variable instead of printing